src/helm/benchmark/scenarios/bhasa_scenario.py

```diff
@@ -0,0 +1,1908 @@
+                neutral_instance = remainder[remainder["label"]==1].iloc[0].to_frame().transpose()
+                data = pd.concat([data, neutral_instance], axis=0, ignore_index=True)
+            for _, row in data.iterrows():
+                passage = "X: " + row["premise"].strip() + "\nY: " + row["hypothesis"].strip()
+                input = Input(passage)
+                output = Output(self.id2label[int(row["label"])])
+                references = [
+                    Reference(output, tags=[CORRECT_TAG]),
+                ]
+                instance = Instance(
+                    input=input,
+                    references=references,
+                    split=self.splits[split]
+                )
+                outputs.append(instance)
+        return outputs
+
+# 1.3 Tamil: IndicXNLI
+class IndicXNLIScenario(Scenario):
+    """
+    IndicXNLI is a Natural Language Inference scenario for 11 Indic languages. The data was
+    automatically translated from the English XNLI dataset into 11 Indic languages using
+    IndicTrans (Ramesh et al., 2021).
+
+    Only the Tamil subset of the data is used in this scenario. The labels are
+    entailment, contradiction and neutral.
+
+    The models are prompted using the following format:
+
+        உங்களுக்கு இரண்டு வாக்கியங்கள், X மற்றும் Y, தரப்படும்.
+        பின்வரும் கூற்றுகளில் எது X மற்றும் Y வாக்கியங்களுடன் மிகப் பொருந்துகிறது எனக் கண்டறியவும்.
+        A: X உண்மை என்றால் Y உம் உண்மையாக இருக்க வேண்டும்.
+        B: X உம் Y உம் முரண்படுகின்றன.
+        C: X உண்மையாக இருக்கும்போது Y உண்மையாக இருக்கலாம் அல்லது இல்லாமல் இருக்கலாம்.
+        A அல்லது B அல்லது C என்ற ஒறே எழுத்தில் மட்டும் பதிலளிக்கவும்.
+
+        X: <premise>
+        Y: <hypothesis>
+        பதில்: <entailment>
+
+        ...
+
+        X: <premise>
+        Y: <hypothesis>
+        பதில்:
+
+    Target completion:
+        <entailment>
+
+    @inproceedings{aggarwal-etal-2022-indicxnli,
+        title = "{I}ndic{XNLI}: Evaluating Multilingual Inference for {I}ndian Languages",
+        author = "Aggarwal, Divyanshu  and
+            Gupta, Vivek  and
+            Kunchukuttan, Anoop",
+        editor = "Goldberg, Yoav  and
+            Kozareva, Zornitsa  and
+            Zhang, Yue",
+        booktitle = "Proceedings of the 2022 Conference on Empirical Methods in Natural Language Processing",
+        month = dec,
+        year = "2022",
+        address = "Abu Dhabi, United Arab Emirates",
+        publisher = "Association for Computational Linguistics",
+        url = "https://aclanthology.org/2022.emnlp-main.755",
+        doi = "10.18653/v1/2022.emnlp-main.755",
+        pages = "10994--11006",
+    }
+    """
+
+    name = "indicxnli"
+    description = "IndicXNLI Natural Language Inference task"
+    tags = ["natural_language_inference"]
+
+    def __init__(self):
+        super().__init__()
+        self.splits = {
+            "train": TRAIN_SPLIT,
```

Comment:
We should use validation split for the in-context examples to be in sync with XNLI.

Reply:
Fixed in latest commit